src/engine/source/api/src/kvdb/handlers.cpp

```diff
@@ -71,8 +72,20 @@ api::Handler managerPost(std::shared_ptr<kvdb_manager::KVDBManager> kvdbManager)
             return ::api::adapter::genericError<ResponseType>("Missing /name");
         }
 
+        if (eRequest.name() == "")
+        {
+            return ::api::adapter::genericError<ResponseType>("Database name is empty");
+        }
+
+        auto exists = kvdbManager->existsDB(eRequest.name());
+
+        if (!exists.has_value())
+        {
+            return ::api::adapter::genericError<ResponseType>("The DB already exists.");
```

Comment:
Sometimes it is used "Database" and others just "DB", please, try to keep the same criteria for the messages